pkg/apis/operator/v1/types.go

```diff
@@ -66,6 +66,12 @@ type InstallationSpec struct {
 	// NodeMetricsPort specifies which port calico/node serves metrics on. If omitted, then metrics are disabled.
 	// +optional
 	NodeMetricsPort *int32 `json:"nodeMetricsPort,omitempty"`
+
+	// FlexVolumePath optionally specifies a custom path for FlexVolume. If not specified, FlexVolume will be
+	// enabled by default. If set to 'None', FlexVolume will be disabled. The default is based on the k8s
+	// provider.
```

Comment:
I think we should have validation on this field which asserts it is either "None", or a valid absolute path. WDYT?

e.g. "path/to/file" is not a valid option since it's a relative path. Neither is "none".